src/toil/cwl/cwltoil.py

```diff
@@ -323,6 +388,17 @@ def makeJob(tool, jobobj, **kwargs):
         wfjob.addFollowOn(followOn)
         return (wfjob, followOn)
     else:
+        # get_requirement
+        resourceReq, _ = tool.get_requirement("ResourceRequirement")
+        if resourceReq:
+            for req in ("coresMin", "coresMax", "ramMin", "ramMax",
+                         "tmpdirMin", "tmpdirMax", "outdirMin", "outdirMax"):
+                r = resourceReq.get(req)
+                if isinstance(r, string_types) and ("$(" in r or "${" in r):
+                    # Found a dynamic resource requirement so use a job wrapper.
+                    job = CWLJobWrapper(tool, jobobj, **kwargs)
```

Comment:
The wrapper has minimal resources coded in the wrapper's super(CWLJobWrapper, self).__init__ method (but you figured that out already).